src/js/components/Video/Video.js

```diff
@@ -254,10 +274,71 @@ const Video = forwardRef(
         onClick: () => showCaptions(caption.active ? -1 : 0),
       }));
 
+      const buttonTable = {
+        closedCaption: captionControls,
+        fullScreen: {
+          onClick: fullscreen,
+        },
+        pause: {
+          onClick: playing ? pause : play,
+        },
+        play: {
+          onClick: playing ? pause : play,
+        },
+        reduceVolume: {
+          a11yTitle: 'video.volumeDown',
+          onClick: volume >= VOLUME_STEP ? quieter : undefined,
+          close: false,
+        },
+        volume: {
+          a11yTitle: 'video.volumeUp',
+          onClick: volume <= 1 - VOLUME_STEP ? louder : undefined,
+          close: false,
+        },
+      };
+
+      let newItem;
+      const controlsArr =
```

Comment:
We try to avoid both abbreviations and type suffixes in variable names. Instead of "controlsArr", I think "controlsMenuItems" will be more descriptive of the contents and its purpose.